libs/design-system/brandmark/src/lib/brandmark.component.html

```diff
@@ -0,0 +1 @@
+<img class="hra-brandmark" [attr.src]="svgPath() | assetUrl" alt="Human Reference Atlas brandmark" />
```

Comment:
Use the `inlineSvg` directive from `ng-inline-svg-2` (already installed) to load the svg